### PR TITLE
Fix for issue #38009 "2D polygon indices barely visible on bright background"

### LINKED ISSUE #38009: 2D Polygon indices are barely visible on bright background
**Godot version:**
3.2.1

**Issue description:**
image
Could be fixed with something like this #36530 probably

### PR DESCRIPTION
**Pull request by levyjonathan31**
Added a black box behind the numbers to make them visible. Also increased font size.
GodotFix1

_bugsquad edit: may fix #38009_

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,47 @@
+name: Build
```
Comment (clayjohn) [inline .github/workflows/build.yml:1]: This file should not be included in the PR

File: sonar-project.properties [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+# This is the name and version displayed in the SonarCloud UI.
+#sonar.projectName=godot
+#sonar.projectVersion=1.0
+
+# Path is relative to the sonar-project.properties file. Replace "\" by "/" on Windows.
+#sonar.sources=.
+
+# Encoding of the source code. Default is default system encoding
+#sonar.sourceEncoding=UTF-8
```
Comment (clayjohn) [inline sonar-project.properties:12]: This should not be included in the PR

**Review by clayjohn — CHANGES_REQUESTED** [review]
The changes you included for sonar projects should not be included in the PR. You;ll have to remove them and update the PR, we have helpful instructions for doing so in the docs here: https://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html#modifying-a-pull-request

When you are finished, there should only be 1 commit remaining

**Comment by clayjohn** [comment]
Also note, [a PR](https://github.com/godotengine/godot/pull/60094) has already been made to fix https://github.com/godotengine/godot/issues/38009. https://github.com/godotengine/godot/pull/60094 has the benefit of using font outlines which is more consistent with what was proposed in #38009 and what is done in other areas of the editor. 

CC @Calinou to weigh in

**Comment by Cretezy** [comment]
It seems like this PR doesn't take https://github.com/godotengine/godot/pull/36530#issuecomment-744750818 into account. I'll give this a shot by porting changes in https://github.com/godotengine/godot/pull/36530/files

**Comment by akien-mga** [comment]
Superseded by #70902.

**Pull request closed without merging**